bin/pipeline.sh

```diff
@@ -37,3 +41,16 @@ pipetask run -j "$1" -b "$2"/butler.yaml \
     --input "$COLLECTION" --output "$FAKES_COLLECTION" \
     --register-dataset-types \
     --qgraph "$FAKES_QGRAPH_FILE"
+
+pipetask qgraph \
```

Comment:
Please add `--long-log` to this command line. There's a long standing bug in ci_hsc_gen3 that we don't turn on long log and so can't get any times or dataId information with a quick glance at the `_build.log`.